app/sae/modules/Core/Model/Translator.php

```diff
@@ -197,19 +198,18 @@ protected static function _getIonicTranslations() {
             foreach ($files as $file) {
                 if (!$file->isFile()) continue;
                 if (pathinfo($file->getPathName(), PATHINFO_EXTENSION) != "csv") continue;
-
+                
                 $resource = fopen($file->getPathName(), "r");
                 while ($data = fgetcsv($resource, 1024, ";", "\"")) {
-                    if (!empty($data[0]) AND !empty($data[1]) AND in_array($data[0], $keys)) {
+                    if (!empty($data[0]) AND !empty($data[1]) AND isSet($flipped_keys[$data[0]]) ) {
```

Comment:
isset and not isSet (probably a case-insensitive FS)
also please make the pull request on the master branch!